errata_tool/cli/erratum.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from errata_tool.erratum import Erratum
+
+USAGE = """
+errata-tool erratum <command>
+
+Sub Commands:
+    get       Get an erratum (advisory)
+"""
+
+
+def get(argv):
+    if not argv:
+        print('errata-tool erratum get <id>')
+        print('specify an advisory ID, eg. "12345"')
+        sys.exit(1)
+    errata_id = argv.pop(0)
+    e = Erratum(errata_id=errata_id)
+    print(e)
+
+
+def main(argv):
+    if not argv or argv[0] == '--help' or argv[0] == '-h':
```

Comment:
argparse whould help here

Reply:
@yazug I spent a lot of time last week trying to do this entirely with argparse, and I went through a couple of iterations, but it's very difficult when you have subcommands that have their own help unique text with examples. For example argparse always wants to handle the "--help" flag in the main parser, and it never passes --help down to the subparsers.

In Ceph we use a full dispatcher https://github.com/alfredodeza/tambo for this type of problem, but I didn't want to add Tambo as a dependency just yet.

Reply:
did you get anywhere with this idea? otherwise we may just want to look at pulling this in as is and make incremental improvements over time.

Reply:
I did not make time to rewrite it all with argparse. :cry: 

I do think that would be a valuable future effort that we could pursue after we merge this PR.